sky/execution.py

```diff
@@ -271,7 +288,7 @@ def launch(
             auto-generate a name.
         retry_until_up: whether to retry launching the cluster until it is
             up.
-        idle_minutes_to_autostop: if provided, the cluster will be auto-stop
+        idle_minutes_to_autostop: if provided, the cluster will be autostop
```

Comment:
keep consistent with cli?

Reply:
Done. Also added the docstr for the other argument as well. I think it may require another PR to consolidate the docstr's for the programmatic API.